server/neptune/temporalworker/job/url_generator.go

```diff
@@ -0,0 +1,33 @@
+package job
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+)
+
+type UrlGenerator struct {
```

Comment:
I've already added support for this:
https://github.com/lyft/atlantis/blob/25d3a061e24f5658a70889d9eed73d1cb81bbc6b/server/neptune/workflows/internal/terraform/state/url.go#L25-L26

and it's being used.